README.md

```diff
@@ -75,37 +82,38 @@ Then follow below steps for building process:
 	./build
 	```
 
-Build the RobotFramework AIO package with all related libraries (defined in `config/repositories/repositories.conf` file).
+Build the RobotFramework AIO package with all related libraries 
+(defined in `config/repositories/repositories.conf` file).
 
-Build script will detect the Operation System (**Windows** or **Linux**) automatically to run the appropriate steps for building installer package.
+Build script will detect the Operation System (**Windows** or **Linux**) 
```

Comment:
I would prefer "operating system" (instead of "Operation System").

Reply:
Hi Holger,

Thank you for your finding.
I have corrected it in the new commit.

Thank you,
Ngoan